### ISSUE TITLE
[Feature Request] browse installed APKs

### ISSUE THREAD

**Issue by nicoursi** [zhanghai/MaterialFiles#192]
It would be nice to allow the possibility to browse installed apks and differentiate them by system APKs and users APKs.
It would be awsome to show the apk icon while browsing. This is very useful because it allows you to locate the right apk without reading the name

On top of that, it would be useful to add options to backup apks and to uninstall them too.

**Comment by zhanghai** [comment]
There are existing app managers available that does a lot of things. The primary reason I'm still considering adding it is that it can allow users to conveniently visit/backup/restore app data directories, but since there's other feature to work on this won't be implemented very soon.

**Comment by nicoursi** [comment]
Unfortunately it's proving very difficult to find a FOSS replacement to ESfileExplorer (version 4.0.2.2). 
Your creation is very nice and has a lot of potential. A good file manager you trust is all you need, really. Very basic app management is a fearure you can easely find in most file managers. 
I understand though that you have different priorities and I really love this file manager!

**Comment by linsui** [comment]
JFYI, Amaze file manager can view, uninstall and backup installed apps.

**Comment by nicoursi** [comment]
Yes, Amaze file manager offers basic app management,  but it doesn't differentiate among system apps and user apps.

**Comment by ildar** [comment]
If that's going to happen then please consider simple backup format that
`oandbackup` uses. It's zip archive + metainfo file.

**Comment by Strikerlulu** [comment]
Please add the app managment so that we can see **backup** and uninstall easily and apks in standard folders

**Comment by zhanghai** [comment]
Also worth considering pulling out APK of install apps, especially [APKS support](https://developer.android.com/studio/command-line/bundletool) for split APKS.

**Comment by PandoraHayami** [comment]
Genuinely, this is genuinely missing
